.eslintrc.json

```diff
@@ -43,9 +44,34 @@
         "no-useless-escape": "off",
         "no-undef": "off",
         "no-prototype-builtins": "off",
-        "local-rules/no-budibase-imports": "error",
+        "local-rules/no-budibase-imports": "error"
+      }
+    },
+    {
+      "files": ["**/*.spec.ts"],
```

Comment:
Can we add the same plugins (except TS) to `spec.js` as well?

Reply:
Good idea. Do you mind if I do that in a follow-up PR, to keep the size down on this one?

Reply:
Yes please, otherwise you will start a new rabbit hole ahah